wonnx-preprocessing/src/image.rs

```diff
@@ -31,7 +31,7 @@ pub fn load_bw_image(
     })
 }
 
-// Loads an image as (1, w, h, 3)
+// Loads an image as (1, 3, w, h)
```

Comment:
This seems inconsistent with what it says below: either this should be `(1, 3, h, w)`, or below should read `from_shape_fn((1, 3, width, height)` where it now says `height, width` instead.